src/components/AttachmentModal.js

```diff
@@ -501,7 +501,6 @@ function AttachmentModal(props) {
                                 report={props.report}
                                 onNavigate={onNavigate}
                                 source={props.source}
-                                onClose={closeModal}
```

Comment:
This was originally in order for the "swipe down to close" functionality, which i extracted to https://github.com/Expensify/App/pull/33869, because it isn't fully implemented yet

Reply:
Got it! Thanks for the explanation!